test/getBuildInfos.js

```diff
@@ -0,0 +1,21 @@
+function getCIInfos() {
+  const { CI_JOB_ID, CI_PIPELINE_ID, CI_COMMIT_SHORT_SHA } = process.env
+  if (CI_PIPELINE_ID && CI_JOB_ID && CI_COMMIT_SHORT_SHA) {
+    return `job: ${CI_JOB_ID} commit: ${CI_COMMIT_SHORT_SHA}`
+  }
+}
+
+function getLocalInfos() {
+  return `${process.env.USER} ${new Date().toLocaleString()}`
+}
+
+module.exports = function getBuildInfos() {
```

Comment:
You removed the LOCAL/CI and the unit/e2e tags. Is this intended?

Reply:
yes, with bs project/build concept it is pretty clear already IMO